packages/@okta/courage-for-signin-widget/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "baseUrl": ".",
+    "outDir": "./target",
+    "sourceMap": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "allowJs": true,
+    "checkJs": false,
+    "skipLibCheck": true,
+    "paths": {
+      "framework/*": ["node_modules/@okta/courage/src/framework/*"],
+      "vendor/*": ["node_modules/@okta/courage/src/vendor/*"],
+      "handlebars-inline-precompile": ["../../../scripts/buildtools/tsc/handlebars-inline-precompile"],
+      "ConfirmationDialog": ["buildtools/ConfirmationDialog.ts"],
```

Comment:
this probably wont work any more as I change the `ConfirmationDialog` to relative path in courage... :(
maybe patch the courage code like copy the empty module to `node_module/courage/...` ?

Reply:
thanks for the heads up, I'll see what I can do

Reply:
I was able to replace it using webpack resolve